pdns/recursordist/rec-main.cc

```diff
@@ -922,6 +928,10 @@ static void loggerBackend(const Logging::Entry& entry)
 {
   static thread_local std::stringstream buf;
 
+  Logger::Urgency u = entry.d_priority ? Logger::Urgency(entry.d_priority) : Logger::Info;
+  if (u > s_logUrgency) {
+    return;
+  }
```

Comment:
Perhaps a comment of what's happening here could be helpful: "We don't need to log anything if the log level is at a higher urgency than the user-requested level".